blueocean-core-js/src/js/components/ResultPageHeader.jsx

```diff
@@ -29,14 +20,7 @@ CloseButton.propTypes = {
 };
 
 export const ResultPageHeader = props => {
-    const {
-        status = 'unknown',
-        title,
-        onCloseClick,
-        startTime,
-        estimatedDurationInMillis,
-        t,
-    } = props;
+    const { status = 'unknown', title, onCloseClick, startTime, estimatedDurationInMillis, t } = props;
```

Comment:
👎

Reply:
well, I guess as long it fits in one line, it is the preferid formatting but I guess that may be configurable.

Reply:
this is an open bug / discussion on this: https://github.com/prettier/prettier/issues/2550